staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -318,12 +319,90 @@ func (e *Store) Create(ctx genericapirequest.Context, obj runtime.Object) (runti
 	return out, nil
 }
 
+func (e *Store) CreateInitialized(ctx genericapirequest.Context, obj runtime.Object) (runtime.Object, error) {
```

Comment:
I think this function is the right place to implement the first half of fail-open: starting a thread to watch for the obj, and removes the top initializer from the list if it's there for too long.

@smarterclayton are you going to do it or do you expect me to do it?